cmd/packer-sdc/internal/fix/README.md

```diff
@@ -0,0 +1,12 @@
+## `packer-sdc fix`
+
+Fix rewrites parts of the plugin codebase to address known issues or common workarounds used within plugins consuming the Packer plugin SDK.
+
+Options:
+
+ -diff    If the -diff flag is set, no files are rewritten. Instead, fix prints the differences a rewrite would introduce.
+
+Available Fixes:
+
+ gocty
+           Adds a replace directive for github.com/zclconf/go-cty to github.com/nywilken/go-cty
```

Comment:
Nice readme! Should the issue be linked here as well?

Reply:
Good call out. I will add that to the README.